```kotlin
import io.sentry.kotlin.multiplatform.Sentry

fun captureError() {
    try {
      throw Exception("This is a test.")
    } catch (e: Exception) {
      Sentry.captureException(e)
    }
}
```
